text/symbols.talon

```diff
@@ -4,6 +4,10 @@ double dash: "--"
 (bracket | brack | left bracket): "{"
 (rbrack | are bracket | right bracket): "}"
 triple quote: "'''"
+(triple grave | triple grave accent):
+    insert("```")
+(grave | grave accent):
+    insert("`")
```

Comment:
I've used the insert function rather than a direct string value. Let me know if you see any issues with this.

Reply:
Now that I see backtick in the punctuation, I imagine these single commands are not needed since the punctuation dictionary covers them